filewriter.py

```diff
@@ -318,7 +318,7 @@ def write(self, time_tag, data):
         
         # Build a template for the file
         if self._counter == 0:
-            self.tagname = "%s_%.0fMHz.ms" % (tstart.datetime.strftime('%Y%m%d_%H%M%S'), self._freq[0]/1e6)
+            self.tagname = "%s_%.0fMHz.ms" % (tstart.datetime.strftime('%Y%m%d_%H%M%S.%f'), self._freq[0]/1e6)
```

Comment:
The standard integration time for slow is 240,000 * 8192 / 196 MHz or 10.03102... s.  For fast the files come out every 10 * 2400 * 8192 / 196 MHz or 1.00310... s.  Maybe truncate at the ms level?

Reply:
will do

Reply:
On second thought, I'll revert this.I shouldn't be doing math on filenames anyway.